modules/apps/static/portal-file-install/portal-file-install-impl/src/main/java/com/liferay/portal/file/install/internal/DirectoryWatcher.java

```diff
@@ -223,7 +225,9 @@ public void run() {
 				}
 
 				synchronized (this) {
-					wait(PropsValues.MODULE_FRAMEWORK_AUTO_DEPLOY_INTERVAL);
+					wait(
+						SystemPropsValues.
+							MODULE_FRAMEWORK_AUTO_DEPLOY_INTERVAL);
```

Comment:
@W2jx @dantewang No need to move this, this is related to fileinstall, not module framework starting.